File: backend/GuguShop.Application/Dto/ProductCreateDto.cs
using Microsoft.AspNetCore.Http;
using System;
using System.Collections.Generic;

namespace GuguShop.Application.Dto
{
    public class ProductCreateDto
    {
        public string Code { get; set; }
        public string Name { get; set; }
        
        public Guid ManufacturerId { get; set; }
        public Guid CategoryId { get; set; }

        public ICollection<Guid> TagIds { get; set; } = new List<Guid>();
        public IFormFile Image { get; set; }
        public Guid? ImageId { get; set; }
    }
}

File: frontend/src/dtos/product.create-dto.ts
import { RcFile, UploadChangeParam, UploadFile } from 'antd/lib/upload/interface';
export interface ProductCreateDto {
    code: string;
    name: string;
    manufacturerId: string;
    categoryId: string;
    tagIds: string[];
    imageRc: UploadChangeParam<UploadFile>;
    image: File;
}
